Jenkinsfile

```
@@ -67,10 +67,11 @@ pipeline {
               } else {
                 eeMainProjectBranch = params.EE_DOWNSTREAM
               }
+              platformVersion = env.JOB_NAME.split('/')[0]
```

Maybe we should add a code comment here that shows the format of the job name. E.g.:

```suggestion
              // Job name e.g.: '7.15/cambpm-ce/cambpm-main/'
              platformVersion = env.JOB_NAME.split('/')[0]
```